app/controllers/questions_controller.rb

```diff
@@ -1,20 +1,32 @@
 class QuestionsController < ApplicationController
+
   def index
-    question=[]
+    @question=[]
```

Comment:
Unnecessary use of instance variable. I'm assuming `@question` is not passed to the template here?